projects/kit/components/range/range.component.ts

```diff
@@ -184,13 +192,13 @@ export class TuiRangeComponent
     @tuiPure
     private computePureKeySteps(
         keySteps: TuiKeySteps | null,
-        min: number,
-        max: number,
+        min: number | null,
+        max: number | null,
```

Comment:
Why is this change required ?

Just pass non-nullable arguments inside this function (actually, you already do it now).

```ts
this.computePureKeySteps(this.keySteps, this.min ?? 0, this.max ?? 100)
```